h2o-jetty-9/build.gradle

```diff
@@ -1,10 +1,10 @@
 dependencies {
     compile project(":h2o-webserver-iface")
-    compile 'org.eclipse.jetty:jetty-server:9.3.20.v20170531'
-    compile "org.eclipse.jetty:jetty-servlets:9.3.20.v20170531"
-    compile 'org.eclipse.jetty:jetty-jaas:9.3.20.v20170531'
-    compile 'org.eclipse.jetty:jetty-proxy:9.3.20.v20170531'
-    compile 'org.eclipse.jetty:jetty-servlet:9.3.20.v20170531'
+    compile 'org.eclipse.jetty:jetty-server:9.4.26.v20200117'
```

Comment:
Just curious: Is there a reason to upgrade the version?

Reply:
No, there is not, besides fixes & possible security fixed on the way: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.26.v20200117

Reply:
In that case because the upgrade includes an API change, I suggest to

- separate this fix from the upgrade,
- put the upgrade only in master - we don't want h2o fix release upgrade to break stuff for users who are embedding h2o into their apps,
- make sure all Hadoop and Spark tests pass.